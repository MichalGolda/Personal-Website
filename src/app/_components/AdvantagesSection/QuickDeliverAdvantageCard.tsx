import AdvantageCard from "./AdvantageCard";
import AdvantageIcon from "./AdvantageIcon";

const CalendarAdvantageIcon = () => (
  <AdvantageIcon src="/calendar.webp" alt="calendar" />
);

const QuickDeliverAdvantageCard = () => (
  <AdvantageCard
    title="Realizacja w określonym terminie"
    content="Szczegółowo planuje etapy pracy dzięki czemu każde rozwiązanie jest gotowe na czas."
    icon={<CalendarAdvantageIcon />}
    data-aos="fade-right"
    data-aos-duration="2000"
  />
);

export default QuickDeliverAdvantageCard;
