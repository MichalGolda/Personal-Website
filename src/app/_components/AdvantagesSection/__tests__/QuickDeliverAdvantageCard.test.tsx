import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import QuickDeliverAdvantageCard from "../QuickDeliverAdvantageCard";

describe("QuickDeliverAdvantageCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<QuickDeliverAdvantageCard />);

    expect(container).toMatchSnapshot();
  });
});
