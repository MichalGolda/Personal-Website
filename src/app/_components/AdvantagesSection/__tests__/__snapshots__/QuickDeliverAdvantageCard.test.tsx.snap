// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuickDeliverAdvantageCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4 text-left justify-left"
    data-aos="fade-right"
    data-aos-duration="2000"
  >
    <img
      alt="calendar"
      class="w-[48px] lg:w-[64px]"
      data-nimg="1"
      decoding="async"
      height="64"
      loading="lazy"
      src="/_next/image?url=%2Fcalendar.webp&w=128&q=75"
      srcset="/_next/image?url=%2Fcalendar.webp&w=64&q=75 1x, /_next/image?url=%2Fcalendar.webp&w=128&q=75 2x"
      style="color: transparent;"
      width="64"
    />
    <h4
      class="font-bold text-lg md:text-xl lg:text-2xl text-secondary"
    >
      Realizacja w określonym terminie
    </h4>
    <p
      class="text-sm md:text-base text-body"
    >
      Szczegółowo planuje etapy pracy dzięki czemu każde rozwiązanie jest gotowe na czas.
    </p>
  </div>
</div>
`;
